v3/how-to-guides/02-pallet-design/a-add-contracts-pallet/index.mdx

```diff
@@ -88,110 +116,196 @@ Implement the Contracts pallet [configuration traits][contracts-config-rustdocs]
 
    ```rust
    // Contracts price units.
-   pub const MILLICENTS: Balance = 1_000_000_000;
-   pub const CENTS: Balance = 1_000 * MILLICENTS;
-   pub const DOLLARS: Balance = 100 * CENTS;
+   // Unit = the base number of indivisible units for balances
+   const UNIT: Balance = 1_000_000_000_000;
+   const MILLIUNIT: Balance = 1_000_000_000;
+   const EXISTENTIAL_DEPOSIT: Balance = MILLIUNIT;
 
    const fn deposit(items: u32, bytes: u32) -> Balance {
-   	items as Balance * 15 * CENTS + (bytes as Balance) * 6 * CENTS
+     (items as Balance * UNIT + (bytes as Balance) * (5 * MILLIUNIT / 100)) / 10
    }
 
    /// We assume that ~10% of the block weight is consumed by `on_initialize` handlers.
    /// This is used to limit the maximal weight of a single extrinsic.
    const AVERAGE_ON_INITIALIZE_RATIO: Perbill = Perbill::from_percent(10);
    ```
 
-1. Add an instance in runtime for `pallet_contracts`.
+1. Add an instance of `pallet_contracts` in the runtime.
 
-   Create an instance of the Contracts pallet in `construct_macro!` inside `runtime/src/lib.rs`:
+   Create an instance of the Contracts pallet in the `construct_macro!` inside `runtime/src/lib.rs`:
 
    ```rust
    /* --snip-- */
-   Contracts: pallet_contracts::{Pallet, Call, Config<T>, Storage, Event<T>},
+   Contracts: pallet_contracts,
    /* --snip-- */
    ```
 
 ## Add API dependencies
 
-The Contracts pallet exposes custom runtime APIs and RPC endpoints
-which enables reading a contracts state from off chain. To use
-the Contracts pallet to make calls to your node's storage without making a transaction, you'll use two other pallets: the Contracts RPC Runtime API pallet and the Contracts RPC pallet.
+The Contracts pallet exposes custom runtime APIs and RPC endpoints.
+
+An example of such an API + RPC is to execute a contract call as a dry-run ‒ so without
+actually modifying any storage.
+This dry-run is often used by off-chain tooling (e.g. a frontend) to display a contracts state.
+Think of an ERC-20 contract, in order to display the balance of an account the frontend
+would execute a dry-run of the balance function via this RPC.
+
+In order to use the Contracts pallet for this you'll use two other pallets: the
+Contracts RPC Runtime API pallet and the Contracts RPC pallet.
```

Comment:
One sentence per line throughout please, we don't use hard wrap on character count.